test_settings.py

```diff
@@ -7,7 +7,10 @@
 
 SECRET_KEY = 'insecure-secret-key'
 
+ROOT_URLCONF = 'csrf.urls'
+
 INSTALLED_APPS = (
+    'csrf.apps.CsrfAppConfig',
```

Comment:
Do we need to declare this elsewhere as well?
Have you confirmed this works in an IDA directly (without using `django.reverse`)?

Reply:
This will need to be add to `INSTALLED_APPS` in all Django services where we want to use the CSRF token endpoint. I have verified it works with LMS.

Reply:
Gotcha.  Thanks.